### PR TITLE
Feature/revsdl 1438 country consent

### PR DESCRIPTION
**Pull request by pvysh9**
Policy manager does not apply "country_consent_passengerRC" to connected non_Primary device after PTU.

### CODE REVIEW

File: src/components/application_manager/src/policies/policy_handler.cc [hunk 1]
```diff
@@ -1470,6 +1470,16 @@ void PolicyHandler::SetRemoteControl(bool enabled) {
   POLICY_LIB_CHECK_VOID();
   policy_manager_->SetRemoteControl(enabled);
 
+  OnCountryConsentChanged(enabled);
```
Comment (bonewell) [inline src/components/application_manager/src/policies/policy_handler.cc]: SetRemoteControl is changing user_consent_passengersRC. I am not sure this call must be here.

Comment (pvysh9) [inline src/components/application_manager/src/policies/policy_handler.cc]: The method OnCountryConsentChanged contains only code from SetRemoteControl after line "policy_manager_->SetRemoteControl" - nothing changed from SetRemoteControl's perspective.

File: src/components/policy/src/policy/src/policy_manager_impl.cc [hunk 2]
```diff
@@ -147,10 +147,20 @@ bool PolicyManagerImpl::LoadPT(const std::string& file,
     listener_->OnCertificateUpdated(*(pt_update->policy_table.module_config.certificate));
   }
 
-
   // Check permissions for applications, send notifications
   CheckPermissionsChanges(pt_update, policy_table_snapshot);
 
+#ifdef SDL_REMOTE_CONTROL
+  // Check country_consent
+  bool has_changed = access_remote_->CheckPTUUpdatesChange(pt_update,
```
Comment (bonewell) [inline src/components/policy/src/policy/src/policy_manager_impl.cc]: Here I see only one question: mobile side will receive notification twice if application has new groups so also. Is it Ok?

File: src/components/policy/src/policy/src/policy_manager_impl.cc [hunk 3]
```diff
@@ -147,10 +147,20 @@ bool PolicyManagerImpl::LoadPT(const std::string& file,
...
+                                        policy_table_snapshot);
+  access_remote_->Init();
```
Comment (bonewell) [inline src/components/policy/src/policy/src/policy_manager_impl.cc]: I wanted avoid use this call. So I replaced country_consent_ filed on country_consent() method. But it isn't enough I see now. :(

File: src/components/application_manager/src/policies/policy_handler.cc (hunk 1 above)
Comment (bonewell) [inline src/components/application_manager/src/policies/policy_handler.cc]: Ok. I see. But name isn't good. Maybe OnRemoteControlChanged ?

File: src/components/policy/src/policy/src/policy_manager_impl.cc (hunk 2 above)
Comment (pvysh9) [inline src/components/policy/src/policy/src/policy_manager_impl.cc]: No, but can be fixed later - when notification about changes in groups works correctly (it doesn't)

**Comment by bonewell** [comment]
Reviewed.

**Pull request merged**